content/en/logs/log_configuration/processors.md

```diff
@@ -96,6 +96,8 @@ If your logs have dates in an attribute that are not in this list, use the log d
 The recognized date formats are: <a href="https://www.iso.org/iso-8601-date-and-time-format.html">ISO8601</a>, <a href="https://en.wikipedia.org/wiki/Unix_time">UNIX (the milliseconds EPOCH format)</a>, and <a href="https://www.ietf.org/rfc/rfc3164.txt">RFC3164</a>.
 </div>
 
+**Note**: As of ISO 8601-1:2019, the basic format is `T[hh][mm][ss]` and the extended format is `T[hh]:[mm]:[ss]`. Earlier versions omitted the T (representing time) in both formats.
```

Comment:
Can you move this down to the list of Notes [below](https://github.com/DataDog/documentation/pull/18039/files#diff-4ccbf274c6ae1131e60c8d9b5624fd0fba6686a757706baaac226a0444578938R105)? Maybe after the first note